lib/internal/http2/core.js

```diff
@@ -937,6 +944,24 @@ function setupHandle(socket, type, options) {
     this[kEncrypted] = false;
   }
 
+  if (isUint32(options.maxSessionInvalidFrames)) {
+    const view = new DataView(this[kNativeFields].buffer);
+    view.setUint32(
+      kSessionMaxInvalidFrames,
+      options.maxSessionInvalidFrames,
+      !isBigEndian
+    );
```

Comment:
I think something like 

```js
const uint32 = new Uint32Array(this[kNativeFields].buffer, kMaxInvalidFrames, 1);
uint32[0] = options.maxSessionInvalidFrames;
```

Basically do the same thing without having to worry about endianness?